src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1193,6 +1194,37 @@ def _configure_security_policy_storage_params(arg_ctx):
                    arg_type=kid_param_type,
                    required=True,)
 
+    #####
+    #           sql managed instance ad-admin
+    ######
+    with self.argument_context('sql mi ad-admin') as c:
+        c.argument('managed_instance_name',
+                   arg_type=managed_instance_param_type)
+
+        c.argument('login',
```

Comment:
Please define arg types (at top of file) that can be used for both sql server and MI arguments, instead of copying

Reply:
Done